SUMO-254673 doc changes for v 1 to v 2 migrations apps (#5019)
SumoLogic/sumologic-documentation: docs/integrations/app-development/gitlab.md

### Step 3: Enable GitLab Event tagging at Sumo Logic

> Sumo Logic needs to understand the event type for incoming events. To enable this, the [x-gitlab-event](https://docs.gitlab.com/ee/user/project/integrations/webhook_events.html#push-events) event type is automatically added to the [Fields](/docs/manage/fields) during installation of the app.

Reviewer: ```suggestion
To properly identify the event type for incoming events, Sumo Logic automatically adds the [x-gitlab-event](https://docs.gitlab.com/ee/user/project/integrations/webhook_events.html#push-events) event type to the [Fields](/docs/manage/fields) during app installation.
```
